compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -403,9 +405,7 @@ void buildAMDAIETransformPassPipeline(OpPassManager &pm) {
   if (clUsePipeline == AIEPassPipeline::PadPackPipeline) {
     auto &modulePassManager = pm.nest<ModuleOp>();
     addMLIRAIRAIELoweringPasses(modulePassManager);
-  } else if (clUsePipeline != AIEPassPipeline::PackPeelPipeline) {
-    assert(false && "Expected either PadPackPipeline or PackPeelPipeline");
-  }
+  } 
```

Comment:
You can remove the if condition as well. In the future we should use `addMLIRAIRAIELoweringPasses` for all pipelines.

Reply:
If you can make this change and rebase on main, then we can merge this PR.